### PR TITLE
Add GET ALL integrations endpoint

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by vova07** [comment]
please review

**Comment by vova07** [comment]
please review

File: spec/components/schemas/Integrations/Integration.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                items:
+                  type: object
+                  properties:
+                    name:
+                      type: string
+                      description: The name of the action
+                    index:
+                      type: integer
+                      minimum: 0
+                      description: The name of the action
```
Comment (ashkarpetin) [inline spec/components/schemas/Integrations/Integration.yaml]:
```suggestion
                      description: The index of the action
```

**Comment by vova07** [comment]
please re-review

**Pull request merged**